src/components/Books.js

```diff
@@ -43,46 +44,47 @@ const Books = () => {
   };
 
   return (
-    <>
+    <div className="container">
       <ul className="books-list">
         {books.map((book) => (
-          <li key={book.item_id} className="py-3 d-flex justify-content-between">
-            <div className="d-flex flex-column">
-              <span>{book.category}</span>
-              <span>{book.title}</span>
-              <span>Anonymous</span>
-              <div>
-                <button type="button">Comment</button>
-                <button type="button" id={book.item_id} onClick={removeFromStore}>Remove</button>
-                <button type="button">Edit</button>
+          <li key={book.item_id} className="row book-item ps-2">
+            <div className="d-flex flex-column col-md-5 mt-4">
+              <span className="book-category mb-2">{book.category}</span>
+              <span className="book-title">{book.title}</span>
+              <span className="book-author">Anonymous</span>
+              <div className='mt-3'>
+                <button type="button" className="book-btn me-3">Comment</button>
+                <button type="button" className="book-btn me-3" id={book.item_id} onClick={removeFromStore}>Remove</button>
+                <button type="button" className="book-btn me-3">Edit</button>
               </div>
             </div>
-            <div className="d-flex flex-column">
-              <span>
-                64%
-              </span>
-              <span>Completed</span>
+            <div className="d-flex col-md-3">
+              <img src={progressLogo} alt="Progress" className="progress-logo mt-5 me-3" />
+              <div className="d-flex flex-column">
+                <span className="percent-number mt-5">64%</span>
+                <span className="percent-symbol">Completed</span>
+              </div>
             </div>
-            <div className="d-flex flex-column">
-              <span>CURRENT CHAPTER</span>
-              <span>Chapter 1</span>
-              <button type="button">UPDATE PROGRESS</button>
+            <div className="d-flex flex-column col-md-4 align-items-start mt-4">
+              <span className="current-chapter">CURRENT CHAPTER</span>
+              <span className="chapter mt-1">Chapter 1</span>
+              <button type="button" className="update-btn mt-4">UPDATE PROGRESS</button>
             </div>
           </li>
         ))}
       </ul>
       <hr />
```

Comment:
[OPTIONAL] - Your margin is set to 1rem (top and bottom).

- [ ] You can consider adding a bit more margin as it is in the Zeplin (2.5rem from top and 1.8rem at the bottom).